nautobot/extras/models/roles.py

```diff
@@ -47,13 +47,13 @@ class RoleModelMixin(models.Model):
     Abstract base class for any model which may have roles.
     """
 
-    role = RoleField()
+    role = RoleField(null=True, blank=True)
```

Comment:
```suggestion
    role = RoleField()
```
`blank` and `null` are set to be `True` by default, Why setting it here again?

Reply:
I was just being explicit. I think perhaps fixing the inheritance is enough to alleviate the reported issue.